src/utils/JSONUtils.ts

```
@@ -29,15 +29,17 @@ export class JSONUtils {
-
-    return Object.keys(obj).reduce((acc: acc, k) => {
-      const pre: any = prefix.length ? prefix + '.' : '';
-      if (typeof obj[k] === 'object' && !Array.isArray(obj[k]))
+    return Object.keys(obj).reduce((acc: any, k) => {
+      const pre: string = prefix.length ? prefix + '.' : '';
+      if (
+        obj[k] !== null &&
+        typeof obj[k] === 'object' &&
+        !Array.isArray(obj[k])
+      ) {
         Object.assign(acc, this.flattenObject(obj[k], pre + k));
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)